src/components/home/BackButton.vue

```diff
@@ -4,15 +4,19 @@ import router from '@/router'
 import ChevronArrowLeft from '@/assets/icons/chevron-left.svg'
 import UiButton from '@/components/ui/UiButton.vue'
 
-import { useMapStore } from '@/stores/map'
 import { useTraveltimeInteractionStore } from '@/stores/interactionMap'
 
-const mapStore = useMapStore()
 const traveltimeInteractionStore = useTraveltimeInteractionStore()
 
+const props = defineProps({
+  url: {
+    type: String,
+    default: '/home',
+  },
+})
+
 function backButtonClicked() {
-  router.push('/home')
-  mapStore.viewPoint = `home`
```

Comment:
This is not a required change but just a question
Here you delete the viewPoint update but this viewPoint is updated in each View component (HomeView, LineView, ...)
That's why the update is useless here, that's right ?

Reply:
yes, exactly